ansible/group_vars/sonic/vars

```diff
@@ -13,6 +13,7 @@ mellanox_hwskus: [ 'ACS-MSN2700' ]
 
 sensor_hwskus: [ "ACS-S6000", "ACS-MSN2700", "Force10-S6000" ]
 
+cavium_hwskus: [ "AS7512", "XP-SIM" ]
 ## Note:
```

Comment:
Now it used to distinguish vendor platform i.e _when: sonic_hwsku in mellanox_hwskus_ . Afraid to break other vendor configuration.